icu4c/source/data/buildtool/filtration_schema.json

```diff
@@ -0,0 +1,82 @@
+{
+    "$id": "http://unicode.org/icu-filter-schema",
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
I used draft-07 here but draft-04 in the code.  I should probably change this here to be draft-04 to maximize compatibility; draft-07 was not yet in the pip version I was using, and we do not need any of the features beyond draft-04.